### PR TITLE
windows/cd: update Indonesian translation

### PR DESCRIPTION
**Pull request by hamexhanif**
add the 'go to root' description, reorder the translation according to the English version, update some translation details to obey the translation guideline

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/windows/cd.md [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
 - Menuju ke induk direktori dari direktori saat ini:
 
 `cd ..`
 
-- Menuju direktori di dalam suatu drive:
+- Menuju suatu direktori pada drive yang sama:
+
+`cd {{jalan/menuju/direktori}}`
+
+- Menuju direktori tertentu di drive [d] yang berbeda:
```
Comment (reinhart1010) [inline pages.id/windows/cd.md:22]:
```suggestion
- Pergi menuju direktori tertentu di [d]rive yang berbeda:
```

File: pages.id/windows/cd.md [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
 
-> Menampilkan nama atau ganti direktori saat ini.
+> Menampilkan direktori kerja saat ini atau pindah ke direktori lain.
 > Informasi lebih lanjut: <https://learn.microsoft.com/windows-server/administration/windows-commands/cd>.
 
-- Menuju suatu direktori pada drive yang sama:
-
-`cd {{lokasi/ke/direktori}}`
-
 - Menampilkan nama dari direktori saat ini:
```
Comment (reinhart1010) [inline pages.id/windows/cd.md:6]:
```suggestion
- Tampilkan nama dari direktori saat ini:
```
Imperative mood, see #8576

File: pages.id/windows/cd.md [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
 
 `cd`
 
+- Menuju root dari drive saat ini:
```
Comment (reinhart1010) [inline pages.id/windows/cd.md:10]:
```suggestion
- Pergi menuju root dari drive saat ini:
```

File: pages.id/windows/cd.md [hunk 4]
```diff
@@ -1,20 +1,24 @@
...
+
+`cd \`
+
 - Menuju ke induk direktori dari direktori saat ini:
```
Comment (reinhart1010) [inline pages.id/windows/cd.md:14]:
```suggestion
- Pergi menuju induk dari direktori dari saat ini:
```

File: pages.id/windows/cd.md [hunk 5]
```diff
@@ -1,20 +1,24 @@
...
 
 `cd ..`
 
-- Menuju direktori di dalam suatu drive:
+- Menuju suatu direktori pada drive yang sama:
```
Comment (reinhart1010) [inline pages.id/windows/cd.md:18]:
```suggestion
- Pergi menuju suatu direktori pada drive yang sama:
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Hi, I'd like to request these changes, please.

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**